iota/transaction/creation.py

```diff
@@ -253,17 +341,26 @@ def add_transaction(self, transaction):
     def add_inputs(self, inputs):
         # type: (Iterable[Address]) -> None
         """
-        Adds inputs to spend in the bundle.
+        Specifies inputs that can be used to fund transactions that spend IOTAs.
```

Comment:
```suggestion
        Specifies inputs that can be used to fund transactions that spend iotas.
```

(actually, now I'm not sure if we should use `iotas` or `IOTAs` 😅)